src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CalculateWorkRule.cs

```diff
@@ -19,6 +20,15 @@ public override Task RunAsync(RuleContext context)
 
             context.NextWorkRequired = context.ValidationContext.ChainedHeader.GetWorkRequired(context.Consensus);
```

Comment:
Don't need to put it is in the context

Reply:
I didn't understand.
The next work is not calculated at this point

Reply:
Ahhh removed from the Context gotcha.... are we sure let me check.

Reply:
Yes correct I will take it out